stage/configuration/test_directory_origin.py

```diff
@@ -1122,9 +1122,50 @@ def test_directory_origin_configuration_record_generation_mode(sdc_builder, sdc_
 
 @pytest.mark.parametrize('data_format', ['LOG'])
 @pytest.mark.parametrize('log_format', ['REGEX'])
-@pytest.mark.skip('Not yet implemented')
-def test_directory_origin_configuration_regular_expression(sdc_builder, sdc_executor, data_format, log_format):
-    pass
+@pytest.mark.parametrize('regular_expression',
+                         ['(\S+) (\S+) (\S+) (\S+) (\S+) (.*)', '(\S+)(\W+)(\S+)\[(\W+)\](\S+)(\W+)'])
+def test_directory_origin_configuration_regular_expression(sdc_builder, sdc_executor, data_format,
+                                                           log_format, shell_executor, file_writer, regular_expression):
+    """Check if the regular expression configuration works. Here we consider logs from DC as our test data.
+    There are two interations of this test case. One with valid regex which should parse logs.
+    Another with invalid regex which should produce null or no result.
+    """
+    file_name = 'custom_log_data.log'
+    file_content = '''2019-04-30 08:23:53 AM [INFO] [streamsets.sdk.sdc_api] Pipeline Filewriterpipeline5340a2b5-b792-45f7-ac44-cf3d6df1dc29 reached status EDITED (took 0.00 s).
+    2019-04-30 08:23:57 AM [INFO] [streamsets.sdk.sdc] Starting pipeline Filewriterpipeline5340a2b5-b792-45f7-ac44-cf3d6df1dc29 ...'''
+
+    field_path_to_regex_group_mapping = LOG_FIELD_MAPPING
+
+    try:
+        files_directory = create_file_and_directory(file_name, file_content, shell_executor, file_writer)
+
+        attributes = {'data_format':data_format,
```

Comment:
Nit: there should be spaces after colons in a dictionary (here and below)
```suggestion
        attributes = {'data_format': data_format,
```

Reply:
Done